Sockets/BackEndMonitoringSockets/include/CSocketWrapper.h

```diff
@@ -8,17 +10,20 @@ class CSocketWrapper
 	CSocketWrapper(const CSocketWrapper&) = delete;
 	CSocketWrapper(CSocketWrapper&&) noexcept = delete;
 	~CSocketWrapper() noexcept = default;
+	CSocketWrapper& operator=(CSocketWrapper&&) = delete;
+	CSocketWrapper& operator=(const CSocketWrapper&) = delete;
 
-	[[nodiscard]] std::string Receive(const int client_socket);
-	bool Send(const int client_socket, const std::string& line);
-	bool CanReceiveData(const int socket) const;
+	bool Receive(const CSocket& client_socket, std::string& message);
+	bool Send(const CSocket& client_socket, const std::string& line);
+	bool CanReceiveData(const CSocket& socket) const;
+	bool IsErrorOccurred(const CSocket& socket) const;
 
 private:
-	const size_t c_max_buffer_size = 2000u;
+	static const int c_max_buffer_size = 500u;
```

Comment:
These fields should not be here

Reply:
Which place is better? Is it better to initialize this field through the constructor (probably add this param to the settings file)? But these values should be the same for client and server? Or make a global const field?